API.md

```diff
@@ -30,10 +30,15 @@ Reauthentication can be performed at anytime and will result in a brand new toke
 
 ### Scheme: OpenID Connect (OIDC)
 
-For authenticating with OpenID Connect, there are two methods:
+For authenticating with OpenID Connect, there are three methods, two of which are run as clients:
 
-- Resource Owner Password Credentials Grant
-- Authorization Code Grant
+- Client-Based
+  - Resource Owner Password Credentials Grant
+  - Authorization Code Grant
+- Protected Resource
```

Comment:
You can link each of these to the sections that follow, allowing the reader to jump to the section of interest.

Reply:
Made links, they should work.